plasmapy/physics/transport.py

```diff
@@ -0,0 +1,172 @@
+"""Functions to calculate transport coefficients."""
+
+from astropy import units
+import numpy as np
+from ..utils import check_quantity, check_relativistic
+from ..constants import (m_p, m_e, c, mu0, k_B, e, eps0, pi, h, hbar,
+                         ion_mass, charge_state)
+from .parameters import Debye_length
+from .quantum import deBroglie_wavelength
+
+
+@check_quantity({"n_e": {"units": units.m**-3},
+                 "T": {"units": units.K, "can_be_negative": False},
+                 "V": {"units": units.m/units.s}})
+def Coulomb_logarithm(n_e, T, particles, V=None):
+    r"""Estimates the Coulomb logarithm.
+
+    Parameters
+    ----------
+    n_e : Quantity
+        The electron density in units convertible to per cubic meter.
+
+    T : Quantity
+        Temperature in units of temperature or energy per particle,
+        which is assumed to be equal for both the test particle and
+        the target particle
+
+    particles : tuple
+        A tuple containing string representations of the test particle
+        (listed first) and the target particle (listed second)
+
+    V : Quantity, optional
+        The relative velocity between particles.  If not provided, it
+        is assumed that :math:`\mu V^2 \sim 3 k_B T` where `mu` is the
+        reduced mass.
+
+    Returns
+    -------
+    lnLambda : float or numpy.ndarray
+        An estimate of the Coulomb logarithm that is accurate to
+        roughly its reciprocal.
+
+    Raises
+    ------
+    ValueError
+        If the mass or charge of either particle cannot be found.
+
+
+    Notes
+    -----
+    The Coulomb logarithm is given by
+
+    .. math::
+    \ln{\Lambda} \equiv \ln\left( \frac{b_{max}}{b_{min}} \right)
+
+    where :math:`b_{min}` and :math:`b_{max}` are the inner and outer
+    impact parameters.
+
+    The outer impact parameter is generally accepted to be the Debye
+    length: :math:`b_{min} = \lambda_D` which is a function of electron
+    temperature and electron density.  At distances greater than the
+    Debye length, charges from other particles will be screened out
+    due to electrons rearranging themselves.
+
+    The inner impact parameter is generally thought to be the maximum
+    of :math:`b_\perp`, the angle at which a test particle is
+    deflected by 90 degrees, and the test particle de Broglie wavelength,
+    `\lambda_B`
+
+    If the Coulomb logarithm is of order unity, then the
+    approximations made in Coulomb collision theory are invalid.
+
+    Examples
+    --------
+    >>> from astropy import units as u
+    >>> Coulomb_logarithm(T=1e6*units.K, n_e=1e19*units.m**-3, ('e', 'p'))
+    14.748259780491056
+
+    See also
+    --------
```

Comment:
People tend not to read the code of package functions when there's a neat docstring like this one, so perhaps pointing out that there's a loooot more down there would be a good move.